scripts/kurtosis/network_params.json

```diff
@@ -71,13 +75,13 @@
     ],
     "tx_spammer_params": {
         "tx_spammer_extra_args": [
-            "--txcount=250"
+            "--txcount=1"
         ]
     },
     "custom_flood_params": {
-        "interval_between_transactions": 0.05
+        "interval_between_transactions": 0.1
```

Comment:
Why/how this helps?

Reply:
Decreases the transaction number generated by custom flood so txpool is not growing too much over time. It's something that we still have to keep working on though. You can see the current status on the image: 
![image](https://github.com/NethermindEth/fb-builder/assets/11379770/ee93993e-57eb-48c9-b7b6-52e8a4bcd811)